tasks/setup.py

```diff
@@ -114,6 +117,22 @@ def _install_pulumi(ctx: Context):
             os.environ["PATH"] = f"{os.environ['PATH']}:{path}"
 
 
+# Check if gke-gcloud-auth-plugin is installed and install it if not
+def _install_gcloud_auth_plugin(ctx):
+    res = ctx.run("gcloud components list --format=json --filter 'name: gke-gcloud-auth-plugin'", hide=True)
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
You might need to skip this when on Windows, not sure it works there

Reply:
I think it should work on Windows as well because the `gcloud` cli is available on Windows. And we make sure the CLI is installed even on Windows